src/main/features/dashboard/views/macro/caseStatus.njk

```diff
@@ -154,8 +153,8 @@
     {{ HWF_Part_Remitted() }}
   {% elif (status === ClaimStatus.NO_RESPONSE or status === ClaimStatus.ELIGIBLE_FOR_CCJ) and claim.helpWithFeesNumber and claim.lastEventTriggeredForHwfCase === 'HWFFullRemision' %}
     {{ HWF_Full_Granted() }}
-  {% elif (status === ClaimStatus.NO_RESPONSE or status === ClaimStatus.ELIGIBLE_FOR_CCJ) and claim.helpWithFeesNumber and claim.lastEventTriggeredForHwfCase === 'RecalculateInterest' %}
-    {{ HWF_Intrest_Recalculate() }}
+  {# {% elif (status === ClaimStatus.NO_RESPONSE or status === ClaimStatus.ELIGIBLE_FOR_CCJ) and claim.helpWithFeesNumber and claim.lastEventTriggeredForHwfCase === 'RecalculateInterest' %}
```

Comment:
If this code is no longer required (removed as part of your story), please remove it instead of commenting it.